ansible/roles/ocp-workload-odo-workshop/tasks/remove_workload.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: post_workload Tasks Complete
+  debug:
+    msg: "Pre-Software checks completed successfully - Removed"
+
+- name: Remove odo workshop resources
+  shell: "oc delete \
+  services,routes,deploymentconfigs,imagestreams,secrets,configmaps,serviceaccounts,rolebindings,serviceaccounts,rolebindings,persistentvolumeclaims,pods \
+  --selector 'app={{odo_application_name}}-{{odo_project_name}},spawner=learning-portal'"
```

Comment:
use YAML multiline:

```yaml
- name: Remove odo workshop resources
  shell: >-
    oc delete
    services,routes,deploymentconfigs,imagestreams,secrets,configmaps,serviceaccounts,rolebindings,serviceaccounts,rolebindings,persistentvolumeclaims,pods
    --selector 'app={{odo_application_name}}-{{odo_project_name}},spawner=learning-portal'
```

Reply:
Thanks! I didn't even know that was possible.